libraries/rtos/rtx/TARGET_CORTEX_M/RTX_Conf_CM.c

```diff
@@ -168,6 +168,12 @@
 #elif defined(TARGET_STM32F401VC)
 #    define OS_CLOCK       84000000
 
+#elif defined(TARGET_NZ32ST1L)
+#    define OS_CLOCK       32000000
+
+#elif defined(TARGET_STM32L152RE)
```

Comment:
A reply to my multiple emails would be much appreciated.

On 24/02/2015 10:26 PM, Martin Kojtal wrote:
>
> In libraries/rtos/rtx/TARGET_CORTEX_M/RTX_Conf_CM.c 
> <https://github.com/mbedmicro/mbed/pull/928#discussion_r25245370>:
>
> > @@ -168,6 +168,12 @@
> >  #elif defined(TARGET_STM32F401VC)
> >  #    define OS_CLOCK       84000000
> >
> > +#elif defined(TARGET_NZ32ST1L)
> > +#    define OS_CLOCK       32000000
> > +
> > +#elif defined(TARGET_STM32L152RE)
>
> This pull request is for NZ32 target, and there are some 
> TARGET_STM32L152RE, can you elaborate?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mbedmicro/mbed/pull/928/files#r25245370>.
>